server/svix-server/src/error.rs

```diff
@@ -34,34 +34,34 @@ pub struct Error {
 
 impl Error {
     #[track_caller]
-    pub fn generic(s: impl fmt::Display) -> Self {
+    pub fn generic(s: impl fmt::Debug) -> Self {
```

Comment:
@svix-aaron1011, is literally doing the opposite at the moment internally. 😂

Reply:
I had noticed a trend towards logging Debug representations, so I went with @svix-gabriel 's suggestion. I'm not sure it gains us a lot of information, but I also don't think it hurts us.

Before I go changing a bunch of call sites, do we want to go with Debug or Display formatting by default for these messages?

My (obviously pretty weak) opinion would be to go with Display and leave more complicated formatting up to the caller, but that does make some places kinda chunky.

Reply:
Looking at @svix-aaron1011's PR, it seems like Debug-by-default does hurt us in some contexts. I'm going to undo that commit unless there are dissenting opinions.

Reply:
😂 - I like the detail of `Debug` representations (and occasionally I find them more useful), but I understand the argument for `Display`.

So I won't push back either way